code/lib/core-common/src/utils/validate-configuration-files.ts

```diff
@@ -22,7 +22,7 @@ export async function validateConfigurationFiles(configDir: string) {
   if (!mainConfigPath) {
     throw new Error(dedent`
       No configuration files have been found in your configDir (${path.resolve(configDir)}).
-      Storybook needs "main.js" file, please add it.
+      Storybook needs "main.js" file, please add it (or pass a custom config dir flag to Storybook to tell where your main.js file is located at).
```

Comment:
@yannbf Do we still need this now that there is the global pre-check? (applies to every instance of this line in this PR)

Reply:
Yes, as `validate-configuration-files` can run in a non-automigrate scenario (anywhere where loadMainConfig is called)